geoips/commandline/geoips_command.py

```diff
@@ -573,12 +573,12 @@ class CommandClassFactory:
         * ...
         * GeoipsListSingleInterfaceTitleFormatters
 
-    * GeoipsGetInterface
+    * GeoipsDescribeArtifact
 
-        * GeoipsGetInterfaceAlgorithm
-        * GeoipsGetInterfaceColormapper
+        * GeoipsGetArtifactAlgorithm
+        * GeoipsGetArtifactColormapper
         * ...
-        * GeoipsGetInterfaceTitleFormatter
+        * GeoipsGetArtifactTitleFormatter
```

Comment:
Should these also be GeoipsDescribeArtifact?  Not GeoipsGet?

Reply:
Yes; Good catch! Fixed this now.